common/djangoapps/entitlements/migrations/0003_auto_20171120_1432.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('entitlements', '0002_auto_20171102_0719'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='courseentitlement',
+            name='uuid',
+            field=models.UUIDField(default=uuid.uuid4, unique=True, editable=False),
```

Comment:
Is there any chance that this could cause issues, if there is data in the database already that isn't unique?

Reply:
There is nothing the Database currently. We do not have entitlements in prod.

Reply:
Also this needs to be updated once Jeff's PR merges to masted.

Reply:
I've think there might be data on stage, so we may want to double check that it is valid before this runs.

Reply:
Will this have negative impacts on open source at all? Just because we don't have data in Prod and can drop data in stage at will, doesn't mean another group might have that option. Unless this feature isn't "released" yet and no one's supposed to be using it?

Reply:
I think if this was a field in the database for a long period of time I would be more concerned, but no one is using it currently.  Unless someone has manually entered a field in the Database on Stage then we should not have a non unique entry. This is why we need to make this change now.  I will take a look at the read replica on Stage before merging.